tests/foreman/installer/test_installer.py

```diff
@@ -1413,7 +1413,14 @@ def test_capsule_installation(sat_default_install, cap_ready_rhel, default_org):
         3. health check runs successfully
 
     :CaseImportance: Critical
+
+    :BZ: 1984400
+
+    :customerscenario: true
     """
+    # Setup HTTP Proxy to add coverage for BZ#1984400
+    sat_default_install.update_setting('http_proxy', settings.http_proxy.un_auth_proxy_url)
```

Comment:
`update_setting` sets the new setting value and returns the current one, which you can later use to restore the original value in teardown. But atm for this test, it should be ok to set http_proxy value to none.

Reply:
I'd recommend using indirect parametrization with setting_update fixture here instead, which will set the setting at start and reset to default at last in teardown, 
`@pytest.mark.parametrize('setting_update', [f'http_proxy =settings.http_proxy.un_auth_proxy_url'], indirect=True)`